src/orchard/inspect.clj

```diff
@@ -9,12 +9,20 @@
 
   Pretty wild, right?"
   (:require
-   [clojure.string :as s])
+   [clojure.string :as s]
+   [orchard.misc :as misc])
   (:import
    (java.lang.reflect Field Modifier)
    (java.util List Map)
    clojure.lang.Seqable))
 
+;; Datafy and Nav are only available since Clojure 1.10
```

Comment:
```suggestion
(require 'clojure.core.protocols)
;; Datafy and Nav are only available since Clojure 1.10
```

...it's just a pedantic touch, but without that require, `resolve` isn't guaranteed to succeed